src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/ReferredOffenderBooking.java

```diff
@@ -22,6 +23,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @EqualsAndHashCode(of = {"referredOffenderBookingId"})
+@ToString(exclude = "offenderDeletionReferral") // to avoid circular reference
```

Comment:
How come you don't need an @JsonIgnore for that circular reference on the column?

Reply:
Because it's not a DTO!  :man_facepalming: